interface/patient_file/reminder/clinical_reminders.php

```diff
@@ -76,7 +76,7 @@
 
   <div class="tab h-auto" style="width: 97%;">
     <div id='report_results'>
-      <table>
+      <table class="table table-bordered table-active table-hover">
```

Comment:
Surround table with a div element that has a class of `table-responsive` ([reference](https://getbootstrap.com/docs/4.4/content/tables/#responsive-tables))

Reply:
on it, thank you for the review :)